.github/CONTRIBUTING.md

```diff
@@ -382,7 +382,7 @@ Once you sign a CLA, all your existing and future pull requests will have the st
 [testing-guidelines]: ../docs/testing-guidelines/testing-guidelines.md
```

Comment:
@bergmeister, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Common Tests - Validate Markdown Files.Should not have errors in any markdown files
```powershell
Expected strings to be the same, but they were different.
Expected length: 9
Actual length:   272
Strings differ at index 0.
Expected: '--EMPTY--'
But was:  '/home/vsts...'
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown.tests.ps1: line 96
96:         $markdownErrors -join "`n" | Should -BeExactly "--EMPTY--"
```